kernel/kernel-api/src/main/java/io/delta/kernel/internal/actions/AddFile.java

```diff
@@ -41,4 +45,16 @@ public class AddFile {
 
     public static final StructType SCHEMA_WITH_STATS = SCHEMA_WITHOUT_STATS
         .add(JSON_STATS_FIELD);
+
+    /**
+     * Full schema of the {@code add} action in the Delta Log.
+     */
+    public static final StructType FULL_SCHEMA = SCHEMA_WITHOUT_STATS
+            .add("stats", StringType.STRING, true /* nullable */)
+            .add(
+                    "tags",
```

Comment:
nit: indentation? or is this intentional?

Reply:
We have checkstyle issues for indentation when wrapping around for function declaration and function callers. We want method parameter declaration indent to be 8, but the function caller multi-line arguments to be 4. Unfortunately in IntelliJ there is no way to distinguish between the two and it always the same for both.

Once 3.2 is released, need to spend sometime on the checkstyle management.